### PR TITLE
1-add-L-block

### LINKED ISSUE #1: Add L block
## User story

```
As a player,
I want to see the "L" block appearing during the game,
So that I can fill an L-like gap with it.
```
## Acceptance criteria

1. The block should be presented on the playfield as:
   ```
   #
   #
   ##
   ```
1. The block is served along with other blocks during the game.

### PR DESCRIPTION
**Pull request by deetree**
Add L block to the playground

Closes #1

### CODE REVIEW

**Comment by deetree** [comment]
Yes, now I know it shouldn't be here for now. Don't be mad at me. I'm still learning 😉

File: src/test/java/com/epam/prejap/teatrees/block/LBlockTest.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+/**
+ * LBlock tests
+ *
+ * @author Mariusz Bal
+ */
```
Comment (marcin-ogorzalek-EPAM) [inline src/test/java/com/epam/prejap/teatrees/block/LBlockTest.java:12]: You know that this is unnecessary by convention?

Comment (marcin-ogorzalek-EPAM) [reply src/test/java/com/epam/prejap/teatrees/block/LBlockTest.java:12]: Of course in tests

**Review by marcin-ogorzalek-EPAM — CHANGES_REQUESTED** [review]
one thing for removal

**Review by marcin-ogorzalek-EPAM — APPROVED** [review]

**Pull request merged**